bigbluebutton-html5/imports/api/2.0/users/server/modifiers/userEjected.js

```diff
@@ -0,0 +1,53 @@
+import { check } from 'meteor/check';
+import Logger from '/imports/startup/server/logger';
+import Users from '/imports/api/2.0/users';
+import VoiceUsers from '/imports/api/2.0/voice-users';
+import removeVoiceUser from '/imports/api/2.0/voice-users/server/modifiers/removeVoiceUser';
+
+export default function userEjected(meetingId, userId) {
+  check(meetingId, String);
+  check(userId, String);
+
+  const selector = {
+    meetingId,
+    userId,
+  };
+
+  const modifier = {
+    $set: {
+      ejected: true,
+      connectionStatus: 'offline',
+      listenOnly: false,
+      validated: false,
+      emoji: 'none',
+      presenter: false,
+      role: 'VIEWER',
+    },
+  };
+
+  const user = VoiceUsers.findOne({ meetingId, voiceUserId: userId });
+
+  if (user) {
```

Comment:
It's interesting, my [comment](https://github.com/bigbluebutton/bigbluebutton/pull/4419#discussion_r141960310) is shown as outdated for some reason, even though I just left it.
Just letting you know that the issue is still there.